README.md

```diff
@@ -370,8 +370,8 @@ se.run
 ```ruby
 module MyServer
   def before_run
-    @socket_manager_path = ServerEngine::SocketManager::Server.generate_path
-    @socket_manager_server = ServerEngine::SocketManager::Server.open(@socket_manager_path)
+    @socket_manager_server = ServerEngine::SocketManager::Server.open_auto_path
```

Comment:
I feel the method name `open_auto_path` is a bit unclear and redundant.
I prefer `open(nil)` (`nil` should be the default so that simply `open`), I think it doesn't break backward compatibility.

Reply:
Understood! I will fix it!

Reply:
Fixed.